app/screens/settings/notifications/send_test_notification_notice.tsx

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useMemo, useRef, useState} from 'react';
+import {useIntl} from 'react-intl';
+import {View} from 'react-native';
+
+import {sendTestNotification} from '@actions/remote/notifications';
+import SectionNotice from '@components/section_notice';
+import {useServerUrl} from '@context/server';
+import {isMinimumServerVersion} from '@utils/helpers';
+import {logError} from '@utils/log';
+import {tryOpenURL} from '@utils/url';
+
+const TIME_TO_IDLE = 3000;
+
+type Props = {
+    serverVersion: string;
+}
+
+type ButtonState = 'idle'|'sending'|'sent'|'error';
+
+function onGoToNotificationDocumentation() {
+    tryOpenURL('https://docs.mattermost.com/collaborate/mention-people.html');
```

Comment:
Should this be pointing to a troubleshooting doc? https://www.figma.com/design/DsYt3C56LgPU5H938FnU4y/Notifications-Test-Tool?node-id=1687-34782&t=1mFS4AAq4YX1hKt0-4

@abhijit-singh do you know if this in the queue for Documentation?

Reply:
I've been meaning to create a ticket for this for a while, so I just created it: https://mattermost.atlassian.net/browse/MM-61226

I assigned @abhijit-singh for now, but feel free to assign any other person if needed. I will add also some comments on the ticket about the behavior.